modules/httpcache/httpcache.go

```diff
@@ -19,13 +19,13 @@ import (
 // AddCacheControlToHeader adds suitable cache-control headers to response
 func AddCacheControlToHeader(h http.Header, d time.Duration) {
 	if setting.IsProd {
-		h.Set("Cache-Control", "private, max-age="+strconv.Itoa(int(d.Seconds())))
+		h.Set("Cache-Control", "private, no-transform, max-age="+strconv.Itoa(int(d.Seconds())))
```

Comment:
IMO in most cases, the `Header().Add("Cache-Control")` could be used instead of `Header().Set`, to avoid overwriting or conflicting (although the old code was using `Set`, new code could make it better)

Then if `d` is zero, no need to add this header.

```go
if d != 0 {
    h.Add("Cache-Control", "max-age=" ....)
}
```

Reply:
I think I still prefer to overwrite it, so last call wins.

> Then if d is zero, no need to add this header.

0 is a significant value, we can't omit the header if `d` is 0.

Reply:
I see, now all page responses have the Cache-Control header. Let me take some more time (maybe one day) to think about it.

Reply:
It's pretty harmless I'd say and generally saying, every HTTP response should set a cache preference, as defaults vary across browsers, proxies etc.

Reply:
Yup, it seems fine to use `max-age=0` here, it just means `no-cache`, according to MDN:

```
max-age=0 is a workaround for no-cache, because many old (HTTP/1.0) cache implementations don't support no-cache. Recently browsers are still using max-age=0 in "reloading" — for backward compatibility — and alternatively using no-cache to cause a "force reloading".
```